solr/core/src/java/org/apache/solr/search/facet/FacetModule.java

```diff
@@ -422,7 +422,7 @@ public static int compare(double a, double b, FacetRequest.SortDirection directi
     }
   }
 
-  static class FacetLongMerger extends FacetSortableMerger {
+  public static class FacetLongMerger extends FacetSortableMerger {
     long val;
```

Comment:
```suggestion
    private long val;
```

Reply:
yeah good point, there's not much point making the class public if subclasses can't access the field, I think I need to review these and make them protected if subclasses could reasonably need to access them.